### ISSUE TITLE
Problems aria label should prioritize content over location

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#179717]
Reported by @jooyoungseo, the problems view aria label currently starts with reporting the location then reads the problem. We should do the opposite.

![A screen shot sows the problems panel](https://user-images.githubusercontent.com/29464607/231282297-5d1661e3-8cf5-46e1-84e3-cdacd29eab44.png)

cc @isidorn if there is historical context here

**Comment by isidorn** [comment]
I agree, and I think this makes good sense. Also I think it should be law hanging as the AccessibilityProvider from the Problems tree returns an aria label, and we should just fine tune it.

fyi @sandy081 @lszomoru
